packages/edit-post/src/components/header/index.js

```diff
@@ -58,7 +58,7 @@ function Header() {
 
 	return (
 		<div className="edit-post-header">
-			<FullscreenModeClose />
+			<AdminMenuToggle />
```

Comment:
Can we do the same for the edit-site package (Site builder)? This will help you figure out better how to make these styles generic and work across screens.